src/main/java/net/atos/zac/app/zaken/ZakenRESTService.java

```diff
@@ -365,7 +365,7 @@ public RESTZaak opschortenZaak(@PathParam("uuid") final UUID zaakUUID,
         final Status status = zaak.getStatus() != null ? zrcClientService.readStatus(zaak.getStatus()) : null;
         final Statustype statustype = status != null ? ztcClientService.readStatustype(status.getStatustype()) : null;
         if (restZaakOpschortGegevens.indicatieOpschorting) {
-            assertPolicy(zaak.isOpen() && !isHeropend(statustype) && !zaak.isOpgeschort());
+            assertPolicy(zaak.isOpen() && !isHeropend(statustype) && !zaak.isOpgeschort() && !zaak.getOpschorting().getReden().isEmpty());
```

Comment:
Wanneer een zaak voor de eerste keer wordt opgeschort heeft de opschorting nog geen reden want de zaak is op dat moment niet opgeschort. Deze policy faalt dus. Heb je je aanpassing wel getest?

Reply:
Opgelost _en getest_ in laatste commit.